Marlin/ultralcd.cpp

```diff
@@ -2389,6 +2396,34 @@ void kill_screen(const char* lcd_msg) {
 
     /**
      *
+     * About Printer > Amb Temp & Humidity
+     *
+     */
+
+    #if ENABLED(DHT_ENABLE)
+
+      void lcd_info_dht_menu(){
+        if (lcd_clicked) { return lcd_goto_previous_menu(); }
+        if (DHT_done == false)
+        {
+          DHT dht;
+          dht.setup(DHT_PIN, DHT_TYPE);
+          safe_delay (1800);
+          DHT_temperature = (dht.getTemperature());
+          DHT_humidity = (dht.getHumidity());
+          DHT_done = true;
+        }
+        START_SCREEN();
+          STATIC_ITEM("Ambient");                                                    //Ambient
+          STATIC_ITEM(MSG_INFO_DHT_MENU);                                            //Temp & Humidity
+          STATIC_ITEM("    Temp (C): ", false, true, itostr3left(DHT_temperature));  //Temp (C): 14
+          STATIC_ITEM("    Humidity: ", false, true, itostr3left(DHT_humidity));     //Humidity: 27
+        END_SCREEN();
```

Comment:
I thought about that. But the leading spaces were better alignment than the "center" option did.